stdlib/inspect.pyi

```diff
@@ -421,11 +421,10 @@ class BoundArguments:
 # Classes and functions
 #
 
-# TODO: The actual return type should be list[_ClassTreeItem] but mypy doesn't
-# seem to be supporting this at the moment:
-# _ClassTreeItem = list[_ClassTreeItem] | Tuple[type, Tuple[type, ...]]
-def getclasstree(classes: list[type], unique: bool = False) -> list[Any]: ...
-def walktree(classes: list[type], children: Mapping[type[Any], list[type]], parent: type[Any] | None) -> list[Any]: ...
+_ClassTreeItem: TypeAlias = list[_ClassTreeItem] | list[tuple[type, ...]]
```

Comment:
I would suggest to write it other way around: 
```suggestion
_ClassTreeItem: TypeAlias = list[tuple[type, ...]] | list[_ClassTreeItem]
```

Since this type is recursive, it better reads this way.